indicators/drought/config.py

```diff
@@ -57,29 +57,22 @@ def parameters(self, country):
     # currently downloading from november 2020 till latest date available
     # and only the belowaverage data
     # this can be changed but if downloading all historical data, it can take about half an hour
-    #TODO: decide if want one file per month or one with general name that contains newest data
-    # IRI_URL = f"https://iridl.ldeo.columbia.edu/SOURCES/.IRI/.FD/.NMME_Seasonal_Forecast/.Precipitation_ELR/.prob/F/(Oct%202020)/(Jan%20{NEXT_YEAR})/RANGEEDGES/data.nc"
-    #TODO: change URL back to above to include newest data. Now using November for debugging
-    # IRI_URL = f"https://iridl.ldeo.columbia.edu/SOURCES/.IRI/.FD/.NMME_Seasonal_Forecast/.Precipitation_ELR/.prob/F/(Oct%202020)/(Nov%202020)/RANGEEDGES/data.nc"
     IRI_URL = f"https://iridl.ldeo.columbia.edu/SOURCES/.IRI/.FD/.NMME_Seasonal_Forecast/.Precipitation_ELR/.prob/F/(Mar%202017)/(Jan%20{NEXT_YEAR})/RANGEEDGES/data.nc"
-    # IRI_URL = f"https://iridl.ldeo.columbia.edu/SOURCES/.IRI/.FD/.NMME_Seasonal_Forecast/.Precipitation_ELR/.prob/F/(Jan%202020)/(Jan%20{NEXT_YEAR})/RANGEEDGES/C/(Below_Normal)/RANGEEDGES/data.nc"
-
     IRI_DIR = "IRI"
     #currently it is solely downloading belowaverage data, this can be changed but will make downloading times longer.
-    IRI_NC_FILENAME_RAW = f"IRI_2020{NEXT_YEAR}.nc"
-    IRI_NC_FILENAME_CRS = f"IRI_2020{NEXT_YEAR}_crs.nc"
+    IRI_NC_FILENAME_RAW = f"IRI_2017{TODAY_YEAR}.nc"
+    IRI_NC_FILENAME_CRS = f"IRI_2017{TODAY_YEAR}_crs.nc"
```

Comment:
can we add a comment explaining why we moved away from the 2020 dataset?
also, did you mean to use NEXT_YEAR on lines 63-64?